include/Animation.h

```diff
@@ -384,7 +384,7 @@ class Snake {
         snakeColor = RgbfColor(hsbColor, true);
         goRight = true;
         head = {false, 0, -1};
-        snake.empty();
+        (void)snake.empty();
```

Comment:
Is this silencing a warning?
[`queue.empty()`](https://www.cplusplus.com/reference/queue/queue/empty/) returns whether the queue is empty and has no side effects. So it can simply be removed as the result is not used.
Or maybe you want to empty the queue instead?

Reply:
I deleted the snake.empty(), it still working without any problems. So I see it as resolved.